vertx/pom.xml

```diff
@@ -10,7 +10,9 @@
     </parent>
 
     <artifactId>gctoolkit-vertx</artifactId>
-
+    <name>GCToolKit Vertx</name>
+    <description>Event module for creating discrete event from parsing log files.</description>
```

Comment:
Messaging backplane for connecting GC log parsers and JVM event consumers.

Reply:
Just to be clear, there may be more than one parser active. A line is read from a GC log and published on the message bus, and then the parsers publish JVMEvent's on the message bus (different channels than the log line, of course!) where the events are picked up by the Aggregator's.